lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -43,6 +43,7 @@ export default function ContentSelector({
   const {
     api: { authToken },
     filePicker: {
+      blackboard: { enabled: blackboardEnabled, listFiles: blackboardListFilesApi },
```

Comment:
Are there any differences between the Blackboard files and Canvas files APIs that the frontend needs to be aware of? Going further, could we consider merging them at some point?